jetty-util/src/main/java/org/eclipse/jetty/util/Pool.java

```diff
@@ -452,7 +527,7 @@ public boolean remove()
          * the multiplex count is maxMultiplex and the entry is not closed,
          * false otherwise.
          */
-        boolean tryAcquire()
+        public boolean tryAcquire()
```

Comment:
This doesn't have to be `public`.

Reply:
Ah yes! It needed to be public for strategies... but not now!